tests/unit-core/test-backtrace.c

```diff
@@ -159,14 +159,14 @@ class_backtrace_callback (jerry_backtrace_frame_t *frame_p, /* frame information
   {
     TEST_ASSERT (jerry_backtrace_is_strict (frame_p));
     TEST_ASSERT (location_p->line == 3);
-    TEST_ASSERT (location_p->column == 12);
+    TEST_ASSERT (location_p->column == 14);
```

Comment:
Why did this changed? Was  the column calculation incorrect before?

Reply:
Yes, it was pointed to the equal sign. Does not makes any sense.

Reply:
ok, thx.